diff --git a/ubuntu-build-service/stretch-desktop-armhf/configure b/ubuntu-build-service/stretch-desktop-armhf/configure
old mode 100755
new mode 100644
index ad42bc9..014e7f5
--- a/ubuntu-build-service/stretch-desktop-armhf/configure
+++ b/ubuntu-build-service/stretch-desktop-armhf/configure
@@ -26,6 +26,9 @@ lb config \
  --chroot-filesystem none \
  --compression gzip \
  --debootstrap-options "--variant=minbase --include=apt-transport-https,gnupg" \
+ --mirror-bootstrap "http://mirrors.ustc.edu.cn/debian/" \
+ --parent-mirror-binary "http://mirrors.ustc.edu.cn/debian/" \
+ --mirror-binary "http://mirrors.ustc.edu.cn/debian/" \
  --distribution stretch \
  --gzip-options '-9 --rsyncable' \
  --iso-publisher 'Linaro; http://www.linaro.org/; linaro-dev@lists.linaro.org' \
