diff --git a/ubuntu-build-service/stretch-desktop-armhf/customization/package-lists/linaro.list.chroot b/ubuntu-build-service/stretch-desktop-armhf/customization/package-lists/linaro.list.chroot
index 99c205e..e3ce3b9 100644
--- a/ubuntu-build-service/stretch-desktop-armhf/customization/package-lists/linaro.list.chroot
+++ b/ubuntu-build-service/stretch-desktop-armhf/customization/package-lists/linaro.list.chroot
@@ -56,7 +56,7 @@ gstreamer1.0-tools
 gvfs
 hostapd
 #htop
-#i2c-tools
+i2c-tools
 idlestat
 ifupdown
 initramfs-tools
@@ -160,4 +160,5 @@ desktop-base
 systemd-sysv
 ntfs-3g
 exfat-fuse
-
+v4l-utils
+fswebcam
