diff --git a/arch/arm/mach-rockchip/rk3288/Kconfig b/arch/arm/mach-rockchip/rk3288/Kconfig
index bbab13f304..beabc90786 100644
--- a/arch/arm/mach-rockchip/rk3288/Kconfig
+++ b/arch/arm/mach-rockchip/rk3288/Kconfig
@@ -59,6 +59,15 @@ config TARGET_FENNEC_RK3288
 	  includes on-board eMMC and 2GB of SDRAM. Expansion connectors
 	  provide access to display pins, I2C, SPI, UART and GPIOs.
 
+config TARGET_EM3288_870A
+    bool "EM3288_870A"
+    select BOARD_LATE_INIT
+    help
+      Fennec is a RK3288-based development board with 2 USB ports,
+      HDMI, micro-SD card, audio, WiFi and Gigabit Ethernet. It also
+      includes on-board 16G eMMC and 2GB of SDRAM. Expansion connectors
+      provide access to display pins, I2C, SPI, UART and GPIOs.	  
+	  
 config TARGET_FIREFLY_RK3288
 	bool "Firefly-RK3288"
 	select BOARD_LATE_INIT
@@ -193,6 +202,8 @@ source "board/rockchip/evb_rk3288_rk1608/Kconfig"
 
 source "board/rockchip/fennec_rk3288/Kconfig"
 
+source "board/rockchip/EM3288_870A/Kconfig"
+
 source "board/rockchip/tinker_rk3288/Kconfig"
 
 endif
diff --git a/board/rockchip/EM3288_870A/EM3288_870A.c b/board/rockchip/EM3288_870A/EM3288_870A.c
new file mode 100644
index 0000000000..846deddb80
--- /dev/null
+++ b/board/rockchip/EM3288_870A/EM3288_870A.c
@@ -0,0 +1,8 @@
+/*
+ * (C) Copyright 2016 Rockchip Electronics Co., Ltd
+ *
+ * SPDX-License-Identifier:     GPL-2.0+
+ */
+
+#include <common.h>
+#include <spl.h>
diff --git a/board/rockchip/EM3288_870A/Kconfig b/board/rockchip/EM3288_870A/Kconfig
new file mode 100644
index 0000000000..13ea8f9b41
--- /dev/null
+++ b/board/rockchip/EM3288_870A/Kconfig
@@ -0,0 +1,15 @@
+if TARGET_EM3288_870A
+
+config SYS_BOARD
+	default "EM3288_870A"
+
+config SYS_VENDOR
+	default "rockchip"
+
+config SYS_CONFIG_NAME
+	default "EM3288_870A"
+
+config BOARD_SPECIFIC_OPTIONS # dummy
+	def_bool y
+
+endif
diff --git a/board/rockchip/EM3288_870A/MAINTAINERS b/board/rockchip/EM3288_870A/MAINTAINERS
new file mode 100644
index 0000000000..d8561caf48
--- /dev/null
+++ b/board/rockchip/EM3288_870A/MAINTAINERS
@@ -0,0 +1,6 @@
+EM3288_870A
+M:	Simon Chen <simon@yeacreate.com>
+S:	Maintained
+F:	board/rockchip/EM3288_870A
+F:	include/configs/EM3288_870A.h
+F:	configs/EM3288_870A_defconfig
diff --git a/board/rockchip/EM3288_870A/Makefile b/board/rockchip/EM3288_870A/Makefile
new file mode 100644
index 0000000000..29a8e844bd
--- /dev/null
+++ b/board/rockchip/EM3288_870A/Makefile
@@ -0,0 +1,7 @@
+#
+# (C) Copyright 2016 Rockchip Electronics Co., Ltd
+#
+# SPDX-License-Identifier:     GPL-2.0+
+#
+
+obj-y	+= EM3288_870A.o
diff --git a/include/configs/EM3288_870A.h b/include/configs/EM3288_870A.h
new file mode 100644
index 0000000000..8482826f3b
--- /dev/null
+++ b/include/configs/EM3288_870A.h
@@ -0,0 +1,18 @@
+/*
+ * (C) Copyright 2016 Rockchip Electronics Co., Ltd
+ *
+ * SPDX-License-Identifier:     GPL-2.0+
+ */
+
+#ifndef __CONFIG_H
+#define __CONFIG_H
+
+#include <configs/rk3288_common.h>
+
+#define CONFIG_SYS_MMC_ENV_DEV 0
+
+#define ROCKCHIP_DEVICE_SETTINGS \
+		"stdout=serial,vidconsole\0" \
+		"stderr=serial,vidconsole\0"
+
+#endif
